Stencil/AutomaticSettings.stencil

```diff
@@ -33,8 +33,9 @@ import {{import}}
           keyPath: \{% call generateKeyPath type %}.{{variable.name}}, 
           requiresRestart: {% if variable.annotations.requiresRestart %}true{% else %}false{% endif %},
           sideEffect: {% if variable.annotations.sideEffect %} { {{variable.annotations.sideEffect }} } {% else %}nil{% endif %}, 
-          uniqueIdentifier: "\\{% call generateKeyPath type %}.{{variable.name}}"{% if variable.annotations.range and variable.type.name == "Float" %},
-          range: {{ variable.annotations.range }}
+          uniqueIdentifier: "\\{% call generateKeyPath type %}.{{variable.name}}"{% if variable.annotations.range and variable.type.name == "Float" or variable.annotations.range and variable.type.name == "Int" %},
```

Comment:
seems like a Stencil bug - using `if variable.annotations.range and ( ... or ...)` to check for float or int types actually adds range and step parameters to the code generated by next loop iteration. weird one. but `... and ... or ... and ...` works fine.

Reply:
newest Stencil broke support for `(` `)`, not sure if it worked intentionally before or as a side-effect of something but it definitely is broken now